### PR TITLE
Ob 4579 extend counselling relation

### PR DESCRIPTION
**Pull request by tkuzynow**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

**Comment by vi-sudo** [comment]
Task linked: [OB-4579 Feature: Add 3 option to advice-seeker-type and add advice-seeker-type to matching logic with counseling offices.](https://app.clickup.com/t/6656196/OB-4579)

File: src/main/java/de/caritas/cob/agencyservice/api/admin/service/agency/AgencyAdminFullResponseDTOBuilder.java [hunk 1]
```diff
@@ -56,6 +58,15 @@ private AgencyAdminResponseDTO createAgency() {
     return responseDTO;
   }
 
+  private List<AgencyAdminResponseDTO.CounsellingRelationsEnum> splitToList(String counsellingRelationsAsCommaSeparatedString) {
+    if (counsellingRelationsAsCommaSeparatedString == null) {
+      return Lists.newArrayList();
+    } else {
```
Comment (SheepFromHeaven) [inline src/main/java/de/caritas/cob/agencyservice/api/admin/service/agency/AgencyAdminFullResponseDTOBuilder.java:64]: Not sure if this works in JAva, but I would not put an Else here in JS

File: api/agencyadminservice.yaml [hunk 2]
```diff
@@ -522,6 +522,12 @@ components:
         deleteDate:
           type: string
           example: "2020-09-02T15:53:23"
+        counsellingRelations:
+          type: array
+          items:
+            type: string
+            enum: [ RELATIVE_COUNSELLING, SELF_COUNSELLING, FAMILY_COUNSELLING ]
```
Comment (SheepFromHeaven) [inline api/agencyadminservice.yaml:529]: I am not sure about variable naming here, as I could not discern between RELATIVE and FAMILY counselling

**Review by SheepFromHeaven — APPROVED** [review]
Approved with comments

File: api/agencyadminservice.yaml (hunk 2 above)
Comment (tkuzynow) [reply api/agencyadminservice.yaml:529]: yeah, I agree, let's double check with the customer/PO about the naming
. Asked this question on general channel.

File: src/main/java/de/caritas/cob/agencyservice/api/admin/service/agency/AgencyAdminFullResponseDTOBuilder.java (hunk 1 above)
Comment (tkuzynow) [reply src/main/java/de/caritas/cob/agencyservice/api/admin/service/agency/AgencyAdminFullResponseDTOBuilder.java:64]: well it's a matter of taste ;) in java we also can do a return. With else it's more explicit and therefore easier to understand. It's less likely that somebody amending the code will put the change in the wrong place.

**Pull request merged**